exercises/collatz-conjecture/src/example.c

```diff
@@ -0,0 +1,32 @@
+#include "collatz_conjecture.h"
+#include <stdbool.h>
+
+static bool is_even(int number)
+{
+   if (number % 2) {
+      return false;
+   }
+
+   return true;
+}
+
+int steps(int start)
+{
+   int step = 0;
+   int number = start;
+
+   if (start <= 0) {
+      return ERROR_VALUE;
+   }
+
+   while (number != 1) {
+      step++;
+
+      if (is_even(number)) {
```

Comment:
As a nitpick on the solution - it seems unnecessary to have a separate function for testing if a number is even.

Reply:
I agree